src/Workspaces/VisualBasic/Portable/LanguageServices/VisualBasicSyntaxFactsService.vb

```diff
@@ -625,13 +625,16 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         End Function
 
         Public Function IsMethodLevelMember(node As SyntaxNode) As Boolean Implements ISyntaxFactsService.IsMethodLevelMember
```

Comment:
Hmm,  I guess some specific tests on the behavior of this method would be in order...